examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,197 @@
+     CheckForError(vi, load_specification_response.status)
+
+     load_timing_response = nidigital_client.LoadTiming(nidigital_types.LoadTimingRequest(vi = vi,
+         file_path = os.path.join(dir, "Timing.digitiming")))
+     CheckForError(vi, load_timing_response.status)
+
+     load_apply_levels_and_timing_response = nidigital_client.ApplyLevelsAndTiming(nidigital_types.ApplyLevelsAndTimingRequest(vi = vi,
+         site_list ="",
+         levels_sheet ="",
+         timing_sheet = os.path.join(dir, "Timing.digitiming"),
+         initial_state_high_pins ="",
+         initial_state_low_pins ="",
+         initial_state_tristate_pins =""))
+     CheckForError(vi, load_apply_levels_and_timing_response.status)
+
+     load_pattern_response = nidigital_client.LoadPattern(nidigital_types.LoadPatternRequest(vi = vi,
+         file_path = os.path.join(dir, "Pattern.digipat")))
+     CheckForError(vi, load_pattern_response.status)
+
+     configure_voltage_response = nidigital_client.ConfigureVoltageLevels(nidigital_types.ConfigureVoltageLevelsRequest(vi = vi,
+         channel_list = channelList,
+         vil = vil,
+         vih = vih,
+         vol = vol,
+         voh = voh,
+         vterm = vterm))
+     CheckForError(vi, configure_voltage_response.status)
+     
+     while True:
+        if termSelect == 'h':
+            configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_HIGH_Z))
+            CheckForError(vi, configure_termination_Mode_response.status)
+
+        if termSelect == 'a':
+            configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_ACTIVE_LOAD))
+            CheckForError(vi, configure_termination_Mode_response.status)
+
+            
+            configure_active_load_levels_response = nidigital_client.ConfigureActiveLoadLevels(nidigital_types.ConfigureActiveLoadLevelsRequest(vi = vi,
+                channel_list = channelList,
+                iol = iol,
+                ioh = ioh,
+                vcom = vcom))
+            CheckForError(vi, configure_active_load_levels_response.status)
+
+        if termSelect == 't':
+            configure_termination_mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_VTERM))
+            CheckForError(vi, configure_termination_mode_response.status)
+
+        if(termSelect == 'h' or termSelect == 'a' or termSelect == 't'):
```

Comment:
I'm wondering what is the value of this check?
If the termSelect is none of the 3 values, it will never exit the loop. If it is one of the 3, it always exits.
So it may be better to check for != h/a/t & show a message that the value is invalid & move the 'break' outside the 'if'

Reply:
Removed while loop completely for simplicity and added invalid value message